apps/web/src/locales/en_GB/messages.json

```
@@ -2370,7 +2370,7 @@
     "message": "Licence file"
   },
   "licenseFileDesc": {
-    "message": "Your licence file will be named something like $FILE_NAME$",
+    "message": "Your license file will be named something like $FILE_NAME$",
```

⛏️ Licence is the correct spelling. The license file is the grant, not the action of granting. The US uses "license" for the noun, while the UK uses "licence". ([source](https://www.grammarly.com/blog/licence-license/))

```suggestion
    "message": "Your licence file will be named something like $FILE_NAME$",
```

---

I will get the product team to review this because they wanted it changed